plugin/src/test/java/io/spine/tools/gradle/bootstrap/func/SpineBootstrapPluginTest.java

```diff
@@ -112,16 +111,13 @@ void generateJava() {
     void applyModelCompiler() {
         configureJavaGeneration();
         GradleProject project = this.project.build();
-        BuildResult result = project.executeTask(build);
-
-        assertThat(result.task(generateValidatingBuilders.path())
-                         .getOutcome()).isEqualTo(SUCCESS);
+        project.executeTask(build);
 
         Collection<String> packageContents = generatedClassFileNames();
         IterableSubject assertPackageContents = assertThat(packageContents);
-        assertPackageContents.containsAtLeast("RollerCoasterVBuilder.class",
-                                              "WagonVBuilder.class",
-                                              "AltitudeVBuilder.class");
+        assertPackageContents.containsAtLeast("RollerCoaster.class",
+                                              "Wagon.class",
+                                              "Altitude.class");
```

Comment:
The purpose of this test is to validate that the Model Compiler was applied. Previously, it did so by ensuring that `VBuilder`s had been generated.

However, plain message classes are generated regardless of the Model Compiler.

Please change the verification strategy for this method. For example, the generated `.desc` file can be used.